app/controllers/webhooks_controller.rb

```diff
@@ -114,10 +114,11 @@ def handle_payment_intent_succeeded(
     end
 
     items = JSON.parse(payment_intent.line_items)
-    recipient = payment_intent.recipient
     is_donation = false
     save_payment_intent = false
-    items = items.reject { |item| item['type'] == 'fee' }
+
+    items.reject! { |item| item['item_type'] == 'transaction_fee' }
```

Comment:
You can merge this, but would be great to do a follow up that includes a test with a `transaction_fee` line item

Reply:
and in that test could you also add a test inside of the payment_intent_spec to test the amount on the intent?

Reply:
yup. done! re-review?

Reply:
Oh perf! yeah there is another test in the payment_intent_spec that tests the method "amount" to make sure that the amount of the payment intent is calculated properly, and I wanted to make sure that adding this new item_type doesn't break that